x-pack/plugins/session_view/public/components/process_tree_alert/styles.ts

```diff
@@ -68,15 +68,11 @@ export const useStyles = ({ isInvestigated, isSelected }: StylesDeps) => {
 
     const alertRuleName: CSSObject = {
       ...alertRowItem,
-      maxWidth: '70%',
     };
 
     const alertStatus: CSSObject = {
       ...alertRowItem,
       textTransform: 'capitalize',
-      '&, span': {
```

Comment:
synced with @codearos about that, we can remove interactions from elements that user can't interact with